Cargo.toml

```diff
@@ -17,6 +17,7 @@ build = "build.rs"
 libc = "0.2"
 nom = { version = "2", features = ["verbose-errors"] }
 byteorder = "0.5"
+time = "0.1"
```

Comment:
I think I already tried that solution but had problems with generics, I prefer this too. I'll try harder !